src/main.js

```diff
@@ -214,7 +214,16 @@ function readDiscoveryFeedProvidersInDirectory(directory) {
     let enumerator = null;
     let info = null;
     let providerBusDescriptors = [];
-    let languages = GLib.get_language_names();
+    let languages = [];
+    let contentLanguages = [];
+    let systemLanguages = GLib.get_language_names();
+
+    if (Gio.SettingsSchemaSource.get_default().lookup('com.endlessm.DiscoveryFeed', false)) {
```

Comment:
In the case when the Schema was not available, the Gio.Settings constructor would fail and the program segfault. I tried catching the error but that did not work.

Reply:
I think programs are generally meant to assume that their own schemas are both installed and compiled

Reply:
Yeah, we should remove this; not having the schema installed is not a scenario we're interested in supporting.